aat/src/aat/resources/features/F-103 New DefaultValue Column/S-103.2.td.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_guid_": "S-103.2",
+	"_extends_": "Import_Base_data",
+	"title": "Import Definition file without the DefaultValue column in the CaseEventToFields tab",
+	"specs": [
+		"an active profile in CCD",
+		"contains a missing default value column"
+	],
+  "request": {
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "file",
+          "filePath": "uk/gov/hmcts/ccd/test_definitions/invalid/BEFTA_Master_Definition_Missing_CaseEventToFields_DefaultValue_Column.xlsx"
```

Comment:
This path doesn't sound quite correct - I thought the `invalid` definitions are ones where we expect validation errors. Happy to be corrected though

Reply:
Fixed in: [d439d7](https://github.com/hmcts/ccd-definition-store-api/pull/1111/commits/d439d7655d27627a95440edc5b0fc6a30fd3cab8)